index.ts

```diff
@@ -4,12 +4,14 @@ import { getRuntime } from './src/utils/Utils';
 if (getRuntime() === 'node') {
   // eslint-disable-next-line
   const undici = require('undici');
+  const { ReadableStream } = require('node:stream/web');
   Reflect.set(globalThis, 'fetch', undici.fetch);
   Reflect.set(globalThis, 'Headers', undici.Headers);
   Reflect.set(globalThis, 'Request', undici.Request);
   Reflect.set(globalThis, 'Response', undici.Response);
   Reflect.set(globalThis, 'FormData', undici.FormData);
   Reflect.set(globalThis, 'File', undici.File);
+  Reflect.set(globalThis, 'ReadableStream', ReadableStream);
```

Comment:
```suggestion
try {
  // eslint-disable-next-line
  const { ReadableStream } = require('node:stream/web');
  Reflect.set(globalThis, 'ReadableStream', ReadableStream);
} catch { /* do nothing */ }
```

Alright, we should probably try-catch this just in case someone tries to use the library in older versions (e.g; 15).